### PR TITLE
Add io.github.kotatogram

### PR DESCRIPTION
**Pull request by ilya-fedin**

### CODE REVIEW

**Comment by ilya-fedin** [comment]
bot, build io.github.kotatogram

File: io.github.kotatogram.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
+{
+    "id": "io.github.kotatogram",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "default-branch": "stable",
```
Comment (bilelmoussaoui) [inline io.github.kotatogram.json]:
```suggestion
```
Not needed.

Comment (ilya-fedin) [reply io.github.kotatogram.json]: fixed

**Comment by LeandroStanger** [comment]
bot, build io.github.kotatogram

**Comment by bilelmoussaoui** [comment]
Have you reached the upstream maintainers and see if they were interested in (co-)maintaining their application on Flathub?

File: fix-appdata.patch [hunk 2]
```diff
@@ -0,0 +1,14 @@
+diff --git a/lib/xdg/kotatogramdesktop.appdata.xml b/lib/xdg/kotatogramdesktop.appdata.xml
+index cb8b990ea..3d79e2e0f 100644
+--- a/lib/xdg/kotatogramdesktop.appdata.xml
++++ b/lib/xdg/kotatogramdesktop.appdata.xml
+@@ -34,6 +34,9 @@
+         <keyword>sms</keyword>
+         <keyword>im</keyword>
+     </keywords>
++    <releases>
```
Comment (bilelmoussaoui) [inline fix-appdata.patch]: Ideally this should be part of upstream

File: io.github.kotatogram.json [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+                 "/man",
+                 "/share/man",
+                 "/lib/systemd",
+                 "/include",
+                 "/lib/pkgconfig",
+                 "/lib/cmake",
+                 "*.la", "*.a"],
+    "modules": [
+        {
+            "name": "cmake",
```
Comment (bilelmoussaoui) [inline io.github.kotatogram.json]: Any reason of rebuilding cmake here?

Comment (ilya-fedin) [reply io.github.kotatogram.json]: tdesktop/kptatogram-desktop require cmake 3.16

File: fix-appdata.patch (hunk 2 above)
Comment (ilya-fedin) [reply fix-appdata.patch]: Ideally should :(

**Comment by RadRussianRus** [comment]
@bilelmoussaoui 
Greetings. As a maintainer of https://github.com/kotatogram/kotatogram-desktop, I'm interested in co-maintaining this application package.

**Comment by nedrichards** [comment]
app-id wise, is there any likelihood that there'll be another app that exists in the io.github.kotatogram namespace? whilst I don't particularly like the convention I do wonder if io.github.kotatogram.desktop would be better. Not a blocker.

**Comment by RadRussianRus** [comment]
I'm not planning to make other apps with this ID as base, so io.github.kotatogram will do.

**Pull request closed without merging**